security/openconnect/src/opnsense/mvc/app/models/OPNsense/Openconnect/General.xml

```diff
@@ -47,6 +47,18 @@
             <Type>cert</Type>
             <Required>N</Required>
         </clientcertificate>
+        <tokenmode type="OptionField">
+            <Required>N</Required>
+                <OptionValues>
```

Comment:
Indent is a bit off here

Reply:
just fyi, copy-pasted the above xml format, thought it was off